modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -44,12 +44,40 @@ public class UpgradeCatchAllCheck {
 			companyId, 551, "test", start, end);
 	}
 
+	public void method(
+		long userId, long groupId, long repositoryId, boolean mountPoint,
+		long parentFolderId, String name, String description, boolean hidden,
+		com.liferay.portal.kernel.service.ServiceContext serviceContext) {
```

Comment:
@MicaelleSilvaa this type doesn't need the complete package, please remove it.

Reply:
resolved